src/biogeochem/CNPhenologyMod.F90

```diff
@@ -3106,8 +3111,15 @@ subroutine CNOffsetLitterfall (num_soilp, filter_soilp, &
                          repr_grainc_to_food_thisyr(p,k) = repr_grainc_to_food_thisyr(p,k) &
                              + repr_grainc_to_food_perharv(p,h,k)
                      end if
+                     repr_grainn_to_food_thispool = npool_to_reproductiven(p,k) - repr_grainn_to_seed(p,k)
                      repr_grainn_to_food(p,k) = t1 * reproductiven(p,k) &
                           + npool_to_reproductiven(p,k) - repr_grainn_to_seed(p,k)
```

Comment:
I asked @samsrabin to remind me the diff between "grain to seed" and "grain to food" and I see here that the two terms summed are the reproductive pool. I'm not sure "seed" is the right term for what's left when you subtract food from the reproductive pool, but that's out of scope for this PR.